doc/api/fs.md

```diff
@@ -2562,6 +2562,19 @@ v0.10.
 `fs.unwatchFile`. `fs.watch` should be used instead of `fs.watchFile` and
 `fs.unwatchFile` when possible.
 
+*Note:* When a file being watched by `fs.watchFile()` disappears and reappears,
+then the `previousStat` reported in the second callback event (the file's
+reappearance) will be the same as the `previousStat` of the first callback
+event (its disappearance).
+
+This happens when:
+- the file is deleted, followed by a restore
+- the file is renamed twice - the second time back to its original name
+
+In such cases the reappearance callback will not report the `previousStat` of
+the file with all zeroes. This is the correct and expected behavior for
+`uv_fs_poll_start()`, the libuv function that `fs.watchFile()` uses internally.
```

Comment:
This paragraph can be removed.  The first sentence is redundant and the last sentence won't be relevant to most readers, they won't know or care it's implemented by libuv.

Reply:
@bnoordhuis Done!